### PR TITLE
[res] Add more PyTorch LSTM examples

### PR DESCRIPTION
**Pull request by binarman**
This PR adds more RNN examples in res/PyTorchExamples.

Related issue: #7939 

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]
LGTM

File: res/PyTorchExamples/examples/LSTM-noinit/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        self.op = nn.LSTM(_input_size, _hidden_size, _number_layers)
+
+    def forward(self, inputs):
+        return self.op(inputs[0])
+
+
+_model_ = net_LSTM()
+
+# dummy input for onnx generation
+_dummy_ = [torch.randn(_seq_length, _batch_size, _input_size)]
```
Comment (binarman) [inline res/PyTorchExamples/examples/LSTM-noinit/__init__.py:24]: I just found that this array is not compatible with testing system I added for onnx models =_=

**Comment by binarman** [comment]
@seanshpark
Sorry for disturbing, I changed one of tests to make it look like other tests with one input

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
> Sorry for disturbing, I changed one of tests to make it look like other tests with one input

No need to worry :)

**Pull request merged**